### DISCUSSION TITLE
Eksport af opgaver

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#6] [Questions] [answered]
KF og Rethinker skal til at gennemgå hele backloggen i Volunteering.

I den forbindelse vil jeg lige høre, om I let har mulighed for at lave en form for eksport eller liste over vores Volunteering backlog, som jeg kan tage med over i et spreadsheet?

**Comment by antonlund18** [comment, marked as answer]
Det lader ikke rigtigt til, at GitHub tilbyder en god måde at gøre det på. Kan du evt. bruge denne her? https://github.com/orgs/xooom-community/projects/2/views/1
